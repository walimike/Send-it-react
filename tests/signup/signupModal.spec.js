import React from 'react';
import { shallow } from 'enzyme';

import SignUpModal from "../../src/components/signUp/SignupModal";

describe('SignUpModal', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<SignUpModal />);
    expect(wrapper).toMatchSnapshot();
  });
});