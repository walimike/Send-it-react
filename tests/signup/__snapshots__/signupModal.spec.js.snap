// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpModal should render without crashing 1`] = `ShallowWrapper {}`;
